hs_file_types/models/raster.py

```diff
@@ -91,26 +92,22 @@ def get_html_forms(self, dataset_name_form=True, temporal_coverage=True, **kwarg
                                    cls="btn btn-primary pull-right",
                                    style="display: none;")
 
-            with div(cls="col-lg-6 col-xs-12"):
-                div("{% crispy orig_coverage_form %}")
-            with div(cls="col-lg-6 col-xs-12"):
-                div("{% crispy cellinfo_form %}")
-
-            with div(cls="pull-left col-sm-12"):
-                with div(cls="well", id="variables"):
-                    with div(cls="row"):
-                        div("{% for form in bandinfo_formset_forms %}")
-                        with div(cls="col-sm-6 col-xs-12"):
-                            with form(id="{{ form.form_id }}", action="{{ form.action }}",
-                                      method="post", enctype="multipart/form-data"):
-                                div("{% crispy form %}")
-                                with div(cls="row", style="margin-top:10px;"):
-                                    with div(cls="col-md-offset-10 col-xs-offset-6 "
-                                                 "col-md-2 col-xs-6"):
-                                        button("Save changes", type="button",
-                                               cls="btn btn-primary pull-right btn-form-submit",
-                                               style="display: none;")
-                        div("{% endfor %}")
+                div("{% crispy orig_coverage_form %}", cls="content-block")
```

Comment:
ok, I realize you just did some formatting changes, but this is the first time I've noticed we're using template language inside dominate... It makes me uncomfortable seeing this, but I'm going to have to chew on it a bit to better understand the implications.  @engineSound @cuahsimarko - Do you have any concerns or praises for doing this?

Reply:
I haven't worked with dominate before. I've seen jinja2 approach as well as a fully separated backend/API - ReactJS frontend. I asked in a previous comment what our other options might be here.

Reply:
@engineSound - would you please chime in?

Reply:
The segment of code has many levels. It is challenging to read and maintain. Regardless of what technology is being used here, I would recommend re-engineer this section.